docs/adr/0007-rancher-namespace-strategy.md

```diff
@@ -0,0 +1,42 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [7. Cluster Import Strategy](#7-cluster-import-strategy)
+  - [Context](#context)
+  - [Decision](#decision)
+  - [Consequences](#consequences)
+  - [Sequence Diagram](#sequence-diagram)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# 7. Rancher Namespace Strategy
+
+- Status: proposed
+- Date: 2023-16-11
+- Authors: @Danil-Grigorev
+- Deciders: @richardcase @alexander-demicev @furkatgofurov7 @mjura @salasberryfin
+
+## Context
+
+The feature of Rancher Manager allows you to **import** an existing cluster, however, the resources created for the rancher-turtles to use are spawned across randomly generated namespaces, preventing rancher-turtles from using the namespace scoped permissions for the namespace scoped resources within the Rancher Manager cluster.
+
+## Decision
+
+The operator will take the responsibility of placing Rancher resources in a specific namespace, defined by the CAPI cluster resource namespace.
+
+The namespace will only dictate the designated location of the Rancher Manager cluster resources, such as **provisioning.cattle/v1.Cluster** and **management.cattle/v3.ClusterRegistrationToken**.
```

Comment:
Rancher Turtles at this stage will only create the **provisioning.cattle/v1.Cluster** instance. It will continue to rely on the Rancher machinery to create the **ClusterRegistrationToken**.

Reply:
The current situation is that we create the **provisioningv1.Cluster** and so its fine for us to state we will create this in a specific namespace. 

We rely on the Rancher machinery to create the **ClusterRegistrationToken** and as it currently stands the **ClusterRegistration** token is automatically created in a namespace thats the name of the managementv3.Cluster (that is autogenerated from the provisioningv1.Cluster) . So we will not be creating the **ClusterRegistration** token and so this should be removed from the ADR.

If we change to generate the managementv3.Cluster in the future then this is cluster scoped and not namespace scoped, so we may need to supercede this ADR at a later date.